app/views/layouts/application.html.erb

```diff
@@ -9,6 +9,7 @@
     <%= stylesheet_link_tag 'application', media: 'all', data: { turbo_track: :reload } %>
     <%= javascript_pack_tag 'application', data: { turbo_track: :reload } %>
     <%= yield :head %>
+    <%= stylesheet_pack_tag "application", "data-turbo-track": "reload" %>
```

Comment:
We could change this to:  `data: { turbo_track: :reload }`? to leave it the same as the others